spark-commons/src/test/scala/za/co/absa/spark/commons/implicits/StructTypeImplicitsTest.scala

```diff
@@ -23,6 +23,8 @@ import za.co.absa.spark.commons.test.SparkTestBase
 
 class StructTypeImplicitsTest extends AnyFunSuite with SparkTestBase with JsonTestData {
   // scalastyle:off magic.number
+  val st: StructType = StructType(Seq(StructField("/some/path", StringType, true)))
+  val structTypeEnhancements = StructTypeEnhancements(st)
```

Comment:
You don't need to create instance of implicit classes this explicitly (that's the beauty of it). It's enough ti  import the implicit class (here line 21), and the functions the implicit class defined "magically: appear at the class it extends (`StructType` in this case).
E.g you can use directly:
`st.isOfType[StringType](path)`